tests/UI/campaigns/functional/FO/11_checkout/02_addresses/02_useDifferentAddressForInvoice.ts

```diff
@@ -1,45 +1,33 @@
 // Import utils
 import helper from '@utils/helpers';
-
-// Import test context
 import testContext from '@utils/testContext';
 
-require('module-alias/register');
-
-const {expect} = require('chai');
-
-// Import utils
-const loginCommon = require('@commonTests/BO/loginBO');
+// Import commonTests
+import loginCommon from '@commonTests/BO/loginBO';
 
 // Import pages
 // Import BO pages
-const dashboardPage = require('@pages/BO/dashboard');
-const ordersPage = require('@pages/BO/orders');
-const orderPageCustomerBlock = require('@pages/BO/orders/view/customerBlock');
+import dashboardPage from '@pages/BO/dashboard';
+import ordersPage from '@pages/BO/orders';
+import orderPageCustomerBlock from '@pages/BO/orders/view/customerBlock';
 
 // Import FO pages
-const foHomePage = require('@pages/FO/home');
-const foProductPage = require('@pages/FO/product');
-const foCartPage = require('@pages/FO/cart');
-const foCheckoutPage = require('@pages/FO/checkout');
-const orderConfirmationPage = require('@pages/FO/checkout/orderConfirmation');
+import foHomePage from '@pages/FO/home';
+import foProductPage from '@pages/FO/product';
+import foCartPage from '@pages/FO/cart';
+import foCheckoutPage from '@pages/FO/checkout';
+import orderConfirmationPage from '@pages/FO/checkout/orderConfirmation';
 
 // Import data
-const {Products} = require('@data/demo/products');
-const {PaymentMethods} = require('@data/demo/paymentMethods');
-const CustomerFaker = require('@data/faker/customer');
-const AddressFaker = require('@data/faker/address');
-
-// Create faker data
-const guestData = new CustomerFaker({password: ''});
-const deliveryAddress = new AddressFaker({country: 'France'});
-const invoiceAddress = new AddressFaker({country: 'France'});
+import {PaymentMethods} from '@data/demo/paymentMethods';
+import {Products} from '@data/demo/products';
+import AddressFaker from '@data/faker/address';
+import CustomerFaker from '@data/faker/customer';
 
-// context
-const baseContext = 'functional_FO_checkout_addresses_useDifferentAddressForInvoice';
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
```

Comment:
```suggestion
import type {BrowserContext, Page} from 'playwright';
```

I guess it's only types right? It would be clearer to use the `import type` keyword, would be more readable

Reply:
Oh right. Thanks.